frame/democracy/src/lib.rs

```diff
@@ -528,20 +502,11 @@ pub mod pallet {
 		ValueQuery,
 	>;
 
-	/// Accounts for which there are locks in action which may be removed at some point in the
-	/// future. The value is the block number at which the lock expires and may be removed.
-	///
-	/// TWOX-NOTE: OK ― `AccountId` is a secure hash.
-	#[pallet::storage]
-	#[pallet::getter(fn locks)]
-	pub type Locks<T: Config> = StorageMap<_, Twox64Concat, T::AccountId, T::BlockNumber>;
-
 	/// True if the last referendum tabled was submitted externally. False if it was a public
 	/// proposal.
 	// TODO: There should be any number of tabling origins, not just public and "external"
 	// (council). https://github.com/paritytech/substrate/issues/5322
 	#[pallet::storage]
-	// Usage of ValueQuery was audited by parity and allowed for now.
```

Comment:
Please leave a comment describing why we use ValueQuery

Reply:
The latest substrate democracy pallet changed to ValueQuery, should we add more comments ?

Reply:
Yeah we internally find ValueQuery dangerous, so at least add a comment that it is used because substrate has it.

Reply:
ill update